modules/virt-virtctl-commands.adoc

```diff
@@ -29,70 +29,143 @@ To view a list of global command options that you can use with any `virtctl` com
 $ virtctl options
 ----
 
-The following table contains the `virtctl` commands used throughout the {VirtProductName} documentation.
+// apinnick: Eventually, we might want to break this up into separate modules and add to CLI assembly.
+[id='managing-vms_{context}']
+== Managing VMs and VMIs
 
-.`virtctl` client commands
+You can use `virtctl` to manage VM or VMI states and to migrate a VM.
 
-[width="100%",cols="42%,58%",options="header",]
+.`virtctl` VM and VMI management commands
+[width="100%",cols="1a,2a",options="header"]
 |===
 |Command |Description
 
 |`virtctl start <vm_name>`
-|Start a virtual machine.
+|Start a VM.
 
 |`virtctl start --paused <vm_name>`
-|Start a virtual machine in a paused state. This option enables you to interrupt the boot process from the VNC console.
+|Start a VM in a paused state. This option enables you to interrupt the boot process from the VNC console.
 
 |`virtctl stop <vm_name>`
-|Stop a virtual machine.
+|Stop a VM.
 
 |`virtctl stop <vm_name> --grace-period 0 --force`
-|Force stop a virtual machine. This option might cause data inconsistency or data loss.
+|Force stop a VM. This option might cause data inconsistency or data loss.
 
-|`virtctl pause vm\|vmi <object_name>`
-|Pause a virtual machine or virtual machine instance. The machine state is kept
+|`virtctl pause vm\|vmi <vm_name>`
+|Pause a VM or VMI. The machine state is kept
 in memory.
 
-|`virtctl unpause vm\|vmi <object_name>`
-|Unpause a virtual machine or virtual machine instance.
+|`virtctl unpause vm\|vmi <vm_name>`
+|Unpause a VM or VMI.
 
 |`virtctl migrate <vm_name>`
-|Migrate a virtual machine.
+|Migrate a VM.
 
 |`virtctl restart <vm_name>`
-|Restart a virtual machine.
+|Restart a VM.
+|===
+
+[id='connecting-to-vms-and-vmis_{context}']
+== Connecting to VMs and VMIs
+
+You can use `virtctl` connect to the serial console, expose a port, set a proxy connection, specify a port, and open a VNC connection to a VM.
```

Comment:
You can use `virtctl` connect to the serial console -> You can use `virtctl` to connect to the serial console